docs/guide/install.rst

```diff
@@ -0,0 +1,29 @@
+============
+Installation
+============
+
+Installing PyPI release
```

Comment:
This is so short it feels weird they're headers, can we just make them normal text?

Reply:
Made it bolded text